test/mocha/index.html

```diff
@@ -0,0 +1,25 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <meta charset="utf-8">
+    <link rel="stylesheet" href="../../node_modules/mocha/mocha.css">
+    <script src="../../node_modules/mocha/mocha.js"></script>
+    <script>mocha.setup('bdd')</script>
+    <script src="../../dist/js-reporters.js"></script>
+    <script src="tests.js"></script>
+    <script>
+        window.onload = function () {
+            // var runner = new JsReporters.MochaAdapter(mocha);
```

Comment:
I will, but I need some time. I have to study for my end sems, they're coming up next week.